instrumentation/google.golang.org/grpc/otelgrpc/test/interceptor_test.go

```diff
@@ -583,39 +583,150 @@ func TestStreamClientInterceptorWithError(t *testing.T) {
 	assert.Equal(t, codes.Error, span.Status().Code)
 }
 
-func TestServerInterceptorError(t *testing.T) {
+var serverChecks = []struct {
+	grpcCode grpc_codes.Code
+	spanCode codes.Code
+	name     string
```

Comment:
Would it not be more readable to rename it to `method`?

Reply:
`getSpanFromRecorder` is always called in this test file with the `name` variable. So I wanted to follow this convention
I agree it makes line 696 (`grpc.StreamServerInfo{FullMethod: check.name}`) a bit less readable, but I think consistency matters more here.
Maybe a better change will be to remove the duplicated name + method in the client's tests, but I preferred to avoid refactoring them.